.github/workflows/macos.yml

```diff
@@ -55,44 +53,44 @@ jobs:
       - name: configure
         run: |
           cd _build
-          ../configure --enable-cobc-internal-checks --enable-hardening --prefix /opt/cobol/gnucobol-gcos --exec-prefix /opt/cobol/gnucobol-gcos
+          ../configure --enable-cobc-internal-checks \
+                       --enable-hardening \
+                       --prefix /opt/cobol/gnucobol-gcos \
+                       --exec-prefix /opt/cobol/gnucobol-gcos
```

Comment:
that's a duplicate and should be dropped as it is by default identical to `prefix`